webapp/templates/tools.html

```diff
@@ -34,11 +54,38 @@
 		</ul>
 	</div>
 
+	{# This tab is for the help section that will relay the message to SysAdmins #}
+	<div id="home" class="col s12">
+		<div class='card z-depth-3' style='width: 60%;margin-left:auto;margin-right:auto;margin-top:2%;'>
+			<div id="home" class="card-content">
+				<span class="card-title teal-title">What We Offer</span>
+				<hr class="teal-line" />
+				<span class="card-title">Free server space</span>
+				<p>You can create a free account on one of our servers. What you do with it is all up to you!</p><br />
+				<p>Please do read the `ACCEPTABLE_USAGE_POLICY` file in your home directory. If you are found to be in violation of this, we will stop whatever you are doing and possibly remove your account.</p>
```

Comment:
The back ticks here are supposed to map to something rendered as code in markdown. That stuff should be in <code> </code> HTML tags :)